### PR TITLE
Try to remove mentions of extensions or extensions/v1alpha1.

### PR DESCRIPTION
**Pull request by mml**
Please read carefully.  I'm sure I've missed something.

### CODE REVIEW

File: docs/devel/api_changes.md [hunk 1]
```diff
@@ -336,15 +336,11 @@ If you found that your change accidentally broke clients, it should be reverted.
 
 In short, the expected API evolution is as follows:
 
-* `extensions/v1alpha1` ->
 * `newapigroup/v1alpha1` -> ... -> `newapigroup/v1alphaN` ->
 * `newapigroup/v1beta1` -> ... -> `newapigroup/v1betaN` ->
 * `newapigroup/v1` ->
 * `newapigroup/v2alpha1` -> ...
 
-While in extensions we have no obligation to move forward with the API at all
```
Comment (lavalamp) [inline docs/devel/api_changes.md]: This should say "alpha" and then I think it remains accurate.

**Comment by lavalamp** [comment]
This looks like a huge improvement, thank you.

File: docs/devel/api_changes.md (hunk 1 above)
Comment (lavalamp) [inline docs/devel/api_changes.md]: @mml if you fix this I'll LGTM :)

**Comment by mml** [comment]
@lavalamp PTAL

**Comment by lavalamp** [comment]
LGTM, thanks!

**Comment by lavalamp** [comment]
@mml Why did you close this before it merged?

**Pull request closed without merging**